### PR TITLE
Add(Immunity) Add install instructions for kong.conf

### PR DESCRIPTION
**Pull request by wongstein**
Add kong.conf changes needed to show GUI changes in Kong Manager

### CODE REVIEW

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 1]
```diff
@@ -94,8 +94,14 @@ Then you can bring up collector with:
...
 ```
 
-#### Step 2. Set up the Collector Plugin
-Enable the Collector Plugin using the Admin API:
+#### Step 2. Set up the Collector Plugin and Kong GUI
+Enable Immunity/Brain GUI changes on Kong Manager by editing the kong.env file to have:
+```
+admin_gui_flags={"IMMUNITY_ENABLED":true}
+```
+Then restart Kong to see the changes take affect.
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
Then restart Kong to see the changes take effect.
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 2]
```diff
@@ -94,8 +94,14 @@ Then you can bring up collector with:
 SQLALCHEMY_DATABASE_URI=postgres://{POSTGRES-USER}:{POSTGRES-PASSWORD}@{POSTGRES HOST}:{POSTGRES POST}/collector CELERY_BROKER_URL={REDIS URI} docker-compose -f docker-compose.yml up -d
 ```
 
-#### Step 2. Set up the Collector Plugin
-Enable the Collector Plugin using the Admin API:
+#### Step 2. Set up the Collector Plugin and Kong GUI
+Enable Immunity/Brain GUI changes on Kong Manager by editing the kong.env file to have:
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]: (Not sure if this matters or whether these are equivalent, just going by the PR title.)

```suggestion
Enable Immunity/Brain GUI changes on Kong Manager by editing the kong.conf file to have:
```

Comment (wongstein) [reply app/enterprise/1.3-x/brain-immunity/install-configure.md]: Oh it is kong.conf!!  Not to be confused with kong.env

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 3]
```diff
@@ -94,8 +94,14 @@ Then you can bring up collector with:
 SQLALCHEMY_DATABASE_URI=postgres://{POSTGRES-USER}:{POSTGRES-PASSWORD}@{POSTGRES HOST}:{POSTGRES POST}/collector CELERY_BROKER_URL={REDIS URI} docker-compose -f docker-compose.yml up -d
 ```
 
-#### Step 2. Set up the Collector Plugin
-Enable the Collector Plugin using the Admin API:
+#### Step 2. Set up the Collector Plugin and Kong GUI
```
Comment (lena-larionova) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]: This title change breaks the link in line 52.

**Pull request merged**